2025/shell_scripting/Week 3 - Shell Scripting Challenge/create_user.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+
+echo "Enter your username: "
+read username
+
+
+echo "Enter your password: "
+read -s password
```

Shellcheck (0.10.0)

[warning] password appears unused. Verify use (or export if used externally).

(SC2034)